tool/site/client-react/src/fhir/customExtensions.ts

```diff
@@ -1,78 +1,150 @@
 }
-export const getGroupIdentifierExtension = getExtensionFinder<GroupIdentifierExtension>(URL_GROUP_IDENTIFIER_EXTENSION)
-
-export interface PrescriptionShortFormIdExtension extends Extension {
+interface PrescriptionShortFormIdExtension extends Extension {
   url: "shortForm",
   valueIdentifier: Identifier
 }
 
-export interface PrescriptionLongFormIdExtension extends Extension {
+interface PrescriptionLongFormIdExtension extends Extension {
   url: "UUID",
   valueIdentifier: Identifier
 }
 
+export const getGroupIdentifierExtension = (extensions: Array<Extension>): GroupIdentifierExtension =>
+  getSingleExtension<GroupIdentifierExtension>(extensions, [
+    URL_GROUP_IDENTIFIER_EXTENSION
+  ])
+
 export const URL_CLAIM_SEQUENCE_IDENTIFIER = "https://fhir.nhs.uk/StructureDefinition/Extension-ClaimSequenceIdentifier"
 export interface ClaimSequenceIdentifierExtension extends Extension {
   url: typeof URL_CLAIM_SEQUENCE_IDENTIFIER,
   valueIdentifier: Identifier
 }
-export const getClaimSequenceIdentifierExtension = getExtensionFinder<ClaimSequenceIdentifierExtension>(URL_CLAIM_SEQUENCE_IDENTIFIER)
+export const getClaimSequenceIdentifierExtension = (extensions: Array<Extension>): ClaimSequenceIdentifierExtension =>
+  getSingleExtension<ClaimSequenceIdentifierExtension>(extensions, [
+    URL_CLAIM_SEQUENCE_IDENTIFIER
+  ])
 
 export const URL_CLAIM_MEDICATION_REQUEST_REFERENCE = "https://fhir.nhs.uk/StructureDefinition/Extension-ClaimMedicationRequestReference"
 export interface ClaimMedicationRequestReferenceExtension extends Extension {
   url: typeof URL_CLAIM_MEDICATION_REQUEST_REFERENCE,
   valueReference: Reference
 }
-export const getClaimMedicationRequestReferenceExtension = getExtensionFinder<ClaimMedicationRequestReferenceExtension>(URL_CLAIM_MEDICATION_REQUEST_REFERENCE)
+export const getClaimMedicationRequestReferenceExtension = (extensions: Array<Extension>): ClaimMedicationRequestReferenceExtension =>
+  getSingleExtension<ClaimMedicationRequestReferenceExtension>(extensions, [
+    URL_CLAIM_MEDICATION_REQUEST_REFERENCE
+  ])
 
-export const URL_REPEAT_INFORMATION = "https://fhir.nhs.uk/StructureDefinition/Extension-EPS-RepeatInformation"
-export interface RepeatInformationExtension extends Extension {
+const URL_REPEAT_INFORMATION = "https://fhir.nhs.uk/StructureDefinition/Extension-EPS-RepeatInformation"
+interface RepeatInformationExtension extends Extension {
   url: typeof URL_REPEAT_INFORMATION
   extension: Array<NumberOfRepeatsIssuedExtension | NumberOfRepeatsAllowedExtension>
 }
-export const getRepeatInformationExtension = getExtensionFinder<RepeatInformationExtension>(URL_REPEAT_INFORMATION)
-
 export const URL_NUMBER_OF_REPEATS_ISSUED = "numberOfRepeatsIssued"
-export interface NumberOfRepeatsIssuedExtension extends Extension {
-  url: typeof URL_NUMBER_OF_REPEATS_ISSUED
+interface NumberOfRepeatsIssuedExtension extends Extension {
+  url: "numberOfRepeatsIssued"
```

Comment:
how come we arent using the cost on line above?

Reply:
merge conflict, good spot, resolved